autotest/src/main/kotlin/no/nav/familie/tilbake/e2e/domene/builder/BehandleVilkårsvurderingStegBuilder.kt

```diff
@@ -0,0 +1,64 @@
+package no.nav.familie.tilbake.e2e.domene.builder
+
+import no.nav.familie.tilbake.e2e.domene.dto.Aktsomhet
+import no.nav.familie.tilbake.e2e.domene.dto.AktsomhetDto
+import no.nav.familie.tilbake.e2e.domene.dto.BehandleVilkårsvurderingDto
+import no.nav.familie.tilbake.e2e.domene.dto.GodTroDto
+import no.nav.familie.tilbake.e2e.domene.dto.HentVilkårsvurderingDto
+import no.nav.familie.tilbake.e2e.domene.dto.SærligGrunn
+import no.nav.familie.tilbake.e2e.domene.dto.SærligGrunnDto
+import no.nav.familie.tilbake.e2e.domene.dto.VilkårsvurderingsperiodeDto
+import no.nav.familie.tilbake.e2e.domene.dto.Vilkårsvurderingsresultat
+import java.math.BigDecimal
+
+class BehandleVilkårsvurderingStegBuilder(hentVilkårsvurderingResponse: HentVilkårsvurderingDto,
+                                          vilkårvurderingsresultat: Vilkårsvurderingsresultat,
+                                          aktsomhet: Aktsomhet?,
+                                          særligeGrunner: List<SærligGrunn>,
+                                          beløpErIBehold: Boolean,
+                                          andelTilbakekreves: BigDecimal?,
+                                          beløpTilbakekreves: BigDecimal?,
+                                          tilbakekrevSmåbeløp: Boolean?) {
+
+    private val vilkårsvurderingsperioder: MutableList<VilkårsvurderingsperiodeDto> = mutableListOf()
+
+    init {
+        hentVilkårsvurderingResponse.perioder.forEach {
+            vilkårsvurderingsperioder.add(
+                    VilkårsvurderingsperiodeDto(periode = it.periode,
+                                                vilkårsvurderingsresultat = vilkårvurderingsresultat,
+                                                begrunnelse = "Dette er en automatisk begrunnelse fra Autotest",
+                                                godTroDto = if (vilkårvurderingsresultat == Vilkårsvurderingsresultat.GOD_TRO) {
+                                                    GodTroDto(begrunnelse = "Dette er en automatisk begrunnelse fra Autotest",
+                                                              beløpErIBehold = beløpErIBehold,
+                                                              beløpTilbakekreves = if (beløpErIBehold) beløpTilbakekreves
+                                                                                                       ?: it.feilutbetaltBeløp else BigDecimal.ZERO
+                                                    )
+                                                } else null,
+                                                aktsomhetDto = if (vilkårvurderingsresultat != Vilkårsvurderingsresultat.GOD_TRO && aktsomhet != Aktsomhet.FORSETT) {
+                                                    AktsomhetDto(aktsomhet = aktsomhet!!,
+                                                                 andelTilbakekreves = andelTilbakekreves,
```

Comment:
Samme gjelder for aktsomhetDto

Reply:
Har lagd separat metode for å opprete aktsomhetDto